README.md

```diff
@@ -163,6 +163,44 @@ block([
 }/>
 ```
 
+## Event handling with reanimated nodes
+
+With reanimated new syntax is possible to be used with `Animated.event`. Instead of providing only a mapping from event fields to animated nodes it is allowed to write a function that takes reanimated values map as an input and returns a block (or any other reanimated function) that will be then used to handle the event.
+
+This syntax allows for providing some post-processing for the event data that does not fit well as a dependency of other nodes we connect to `Animated.View` component props.
+[See example](https://github.com/kmagiera/react-native-reanimated/blob/master/Example/movable/index.js)
+```js
+<PanGestureHandler
+  onGestureEvent={event([
+    {
+      nativeEvent: ({ translationX: x, translationY: y, state }) =>
+        block([
+          set(this._transX, add(x, offsetX)), set(this._transY, add(y, offsetY)),
+          cond(eq(state, State.END), [set(this.offsetX, add(this.offsetX, x)), set(this.offsetY, add(this.offsetY, y))]),
+        ]),
+    },
+  ])}
+>
+  <Animated.View
+    style={{ transform: [{ translateX: this._transX, translateY: this._transY }], }}
+  />
+</PanGestureHandler>
+```
+However, it is still fully "declarative" and could be described as some kind of syntactic sugar and does not affect neither complexity nor execution time or cost of evaluation.
+
+It is also available to use function with single field of `nativeEvent`:
```

Comment:
I'd move this example as a first one since it is simpler, then present the above example with the following text:

> If you'd like to use more than one event attribute in your reanimated code it is also supported. Instead of defining event handler method for a single attribute you can define at the level of `nativeEvent`. Here is an example that takes both translation attributes and state attribute from `PanGestureHandler` event: